libs/asset-editor/src/lib/components/asset-editor-tab-general/asset-editor-tab-general.component.html

```diff
@@ -2,6 +2,15 @@
   <form [formGroup]="_form" class="flex flex-row max-h-full">
     <div class="flex flex-column form-column mr-8">
       <div class="flex flex-column bg-white mb-4 py-4 px-6 overflow-y-scroll">
+        <mat-form-field class="mb-2">
```

Comment:
Add a title "Workgroup" like the other areas have, e.g. Thema
Translate the label and title